src/main/java/io/ebean/dbmigration/ddlgeneration/platform/SqlServerDdl.java

```diff
@@ -157,4 +190,28 @@ public void addColumnComment(DdlBuffer apply, String table, String column, Strin
 
     // do nothing for MS SQL Server (cause it requires stored procedures etc)
   }
+  
+  /**
+   * It is rather complex to delete a column on SqlServer as there must not exist any references
+   * (constraints, default values, indices and foreign keys). The list is not yet complete, as
+   * indices over multiple columns will not yet deleted.
+   * (This may be changed to delete all refering objects by using the sys.* tables later)
```

Comment:
Yes, I like the idea of a set of migration stored procedures one of which we could use for 'drop column'